packages/ibm-products/src/components/ConditionBuilder/utils/handleKeyboardEvents.js

```diff
@@ -53,6 +53,12 @@ const excludeKeyPress = (evt) => {
   );
 };
 
+const getVisibleOptions = (parentContainer) => {
+  return Array.from(parentContainer.querySelectorAll(`[role="option"]`))
+    .filter((el) => !el.hasAttribute('aria-disabled'))
+    .filter((el) => !el.hasAttribute('aria-hidden'));
```

Comment:
could this be combined into a single `filter` check? `!el.hasAttribute(a) || !el.hasAttribute(b)`? that way it doesn't has to run through the array twice.